string-manipulation-1/src/is-lower-cased.js

```diff
@@ -0,0 +1,12 @@
+/* exported isLowerCased */
+function isLowerCased(word) {
+  for (let i = 0; i < word.length; i++) {
+    const checker = word.charAt(i);
+    const tester = checker.toLowerCase();
+
+    if (checker !== tester) {
+      return false;
+    }
+  }
+  return true;
+}
```

Comment:
No loop needed here! We can simplify this. You don't need to check each character of the string, we can just check if the whole string is equal to if the whole string was converted to lowercase
```
function isLowerCased(word) {
  return word === word.toLowerCase();
}
```